### PR TITLE
Wbxappapi fixes

### PR DESCRIPTION
**Pull request by marcin-bazyl**
These are 2 commits with fixes for 2 issues I've found today when testing the web app with wbxappapi:
1. SDK fails to get the meeting info for meetings currently in progress when web app is loaded
2. SDK fails to get the meeting info for PMR

TODO:
Tests need to be updated/added.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js [hunk 1]
```diff
@@ -201,7 +201,7 @@ MeetingInfoUtil.getRequestBody = (options) => {
       break;
     case _PERSONAL_ROOM_:
       body.userId = destination;
-      body.orgId = ''; // TODO: when to use the org iD (fetch others PMR)
+      body.orgId = webex.credentials.getOrgId(); // TODO: when to use the org iD (fetch others PMR)
```
Comment (marcin-bazyl) [inline packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:204]: this TODO comment needs updating

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/utilv2.js [hunk 2]
```diff
@@ -72,12 +73,18 @@ describe('plugin-meetings', () => {
...
+
         const res = MeetingInfoUtil.getRequestBody({
           type: _PERSONAL_ROOM_,
-          destination: 'userId_1234'
+          destination: 'userId_1234',
+          webex
         });
 
-        assert.equal(res.orgId, '');
+        assert.equal(res.orgId, 'my-realm');
```
Comment (emanuallan) [inline packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/utilv2.js:87]: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-274567

File: packages/node_modules/@webex/plugin-meetings/test/integration/spec/space-meeting.js [hunk 3]
```diff
@@ -44,6 +44,19 @@ skipInNode(describe)('plugin-meetings', () => {
...
+          alice.webex.meetings.config.experimental.enableUnifiedMeetings = true;
+
+          const destinationWithType = await alice.webex.meetings.meetingInfo.fetchMeetingInfo(space.url, 'CONVERSATION_URL');
+          const destinationNoType = await alice.webex.meetings.meetingInfo.fetchMeetingInfo(space.url);
+
+          assert.exists(destinationNoType);
+          assert.exists(destinationWithType);
+          assert.exists(destinationNoType.body.meetingNumber);
+          assert.exists(destinationWithType.body.meetingNumber);
+
```
Comment (emanuallan) [inline packages/node_modules/@webex/plugin-meetings/test/integration/spec/space-meeting.js:57]: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-274564

Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/test/integration/spec/space-meeting.js:57]: looks good

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js (hunk 1 above)
Comment (lalli-flores) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:204]: @marcin-bazyl can you elaborate on what needs to be updated here?

Comment (marcin-bazyl) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:204]: This is not my todo comment, it was introduced by @arun3528 in https://github.com/webex/webex-js-sdk/pull/2174/ so it's best for Arun to comment, but I think we need to check if this fix that we're doing here works for fetching PMR of people outside of your org. If it does, then the todo comment can be removed. If it doesn't then a jira should be raised to fix that.

**Comment by emanuallan** [comment]
This pr was broken into 2 separate PR's with their corresponding test. This PR can be safely discontinued here and continued in their respective PR.
1. SDK fails to get the meeting info for meetings currently in progress when web app is loaded
https://github.com/webex/webex-js-sdk/pull/2220
2. SDK fails to get the meeting info for PMR
https://github.com/webex/webex-js-sdk/pull/2219

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js (hunk 1 above)
Comment (lalli-flores) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:204]: I confirmed with the API team that the org Id that we need to send should correspond to that of the user for which we are trying to fetch PMR information. This code won't work for external users

Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js:204]: @taymoork2  can you remove this code we can add that in the next one

**Comment by taymoork2** [comment]
Closing in favor of #2231

**Pull request closed without merging**